ext/bcmath/libbcmath/src/compare.c

```diff
@@ -42,34 +42,32 @@
 int _bc_do_compare(bc_num n1, bc_num n2, bool use_sign, bool ignore_last)
 {
 	char *n1ptr, *n2ptr;
+	int result_n1_is_large = -1;
 
 	/* First, compare signs. */
-	if (use_sign && n1->n_sign != n2->n_sign) {
+	if (use_sign) {
 		if (n1->n_sign == PLUS) {
-			/* Positive N1 > Negative N2 */
```

Comment:
Keep the comment?

Reply:
Since this change was not effective in release builds, we decided not to modify this file in this PR.